app-authz-jakarta-servlet/src/test/resources/quickstart-realm.json

```diff
@@ -86,7 +84,7 @@
       "adminUrl": "http://127.0.0.1:8080/authz-servlet",
       "bearerOnly": false,
       "redirectUris": [
-        "http://127.0.0.1:8080/authz-servlet/*"
+        "*"
```

Comment:
Same comment like above for the other file

Reply:
I am unresolving this comment as wildcard is still here? Any reason why it is needed and was not needed before with old adapter?

Reply:
Hope is fine now.